package.json

```diff
@@ -1,23 +1,23 @@
 {
   "name": "eslint-config-mavenlint",
-  "version": "2.1.0",
+  "version": "2.2.0",
   "description": "Mavenlink ESLint config",
   "main": "index.js",
   "repository": "https://github.com/mavenlink/mavenlint.git",
   "license": "MIT",
   "devDependencies": {
-    "eslint": "^3.19.0",
-    "eslint-plugin-import": "^2.6.1",
+    "eslint": "^4.14.0",
+    "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.1.0"
+    "eslint-plugin-react": "^7.5.1"
   },
   "peerDependencies": {
-    "eslint": ">= 3.19.0",
-    "eslint-plugin-import": ">= 2.6.1",
+    "eslint": "^4.14.0",
+    "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": ">= 5.1.1",
-    "eslint-plugin-react": ">= 7.1.0"
+    "eslint-plugin-react": "^7.5.1"
   },
   "dependencies": {
-    "eslint-config-airbnb": "^15.0.2"
+    "eslint-config-airbnb": "15.1.0"
```

Comment:
Figured we're better off locking to a specific version here so that we know exactly what rules we're bringing in.